velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

```diff
@@ -253,6 +265,18 @@ TEST_F(VeloxSubstraitRoundTripPlanConverterTest, avg) {
   assertPlanConversion(plan, "SELECT avg(c4) FROM tmp");
 }
 
+TEST_F(VeloxSubstraitRoundTripPlanConverterTest, arrayLiteral) {
+  auto flatVector = sequence<int32_t>(1);
```

Comment:
No need to define this variable, Call sequence directly.

```
makeRowVector({sequence<int32_t>(1)}),
```

Or, better yet, use empty row vector and remove sequence method.

```
makeRowVector(ROW({}, {}), 1);
```